plugin/advisor/pg/advisor_naming_table.go

```diff
@@ -33,16 +33,19 @@ func (*NamingTableConventionAdvisor) Check(ctx advisor.Context, statement string
 	if err != nil {
 		return nil, err
 	}
-	// Naming length limit is not enabled for PG, the PG parser will auto slice the name to make sure its length <= 63
-	// Reference: https://stackoverflow.com/questions/27865770/how-long-can-postgresql-table-names-be
-	format, _, err := advisor.UnamrshalNamingRulePayloadAsRegexp(ctx.Rule.Payload)
+	// Naming length limit feature is limited for PG, the PG parser will auto slice the name to make sure its length <= 63
```

Comment:
I guess we don't need comment here, the constant in sql_review.go is good enough.